public/css/styles.css

```diff
@@ -1,3 +1,7 @@
+*{
+  text-align:  center; /*I'll never get to do this again in my life*/
+}
```

Comment:
This should be 

```CSS
table {
  margin: 0 auto;
}
```

Reply:
Body does that. * Centers text on all elements on the page. Something I'll never get to do again, ever 😛

Reply:
So hacky LOL